rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/InfoView.java

```diff
@@ -125,7 +127,8 @@ Canvas build() {
         DetailViewerField n3 = new DetailViewerField("nodeProvider", "Owner");
         DetailViewerField n4 = new DetailViewerField("hosts", "Hosts");
         DetailViewerField n5 = new DetailViewerField("nodes", "Nodes");
-        this.nsDetails.setFields(n1, n2, n3, n4, n5);
+        DetailViewerField n6 = new DetailViewerField("additionalInformations", "Additional Informations");
```

Comment:
Additional Informations => Additional Information

Reply:
Additional Informations since we can use this "rm portal section" to display multiple informations of the selected node source (cost, next billing date,..)

Reply:
I agree with @mboussaa . "information" has no plural form. It never takes "s".

Reply:
Ok thx